### DISCUSSION TITLE
Tabler dark & light mode issue

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#1172] [Q&A (Help)] [answered]
I notice that many Backpack elements used with the Tabler template are not very visible, especially in Dark mode!
I think all table headers should have a background and field color that makes them visible and distinguishable from the rows.
<img width="946" alt="BKP6 crud-headers" src="https://github.com/user-attachments/assets/4db8ce3d-088b-4911-8f05-d254eaa14224">

Also all modals do not look like that because their background color is not very different from the background color of the page below.
<img width="870" alt="BKP6-modals" src="https://github.com/user-attachments/assets/081f0c94-e58c-4fe7-9838-45be543194ea">

Can something be done globally?
What does the BKP team think?

**Comment by jcastroa87** [comment, marked as answer]
Hello @blondie63 

This is very similar to default Tabler https://tabler.io/admin-template/preview

Of course each user will have their personal preferences.

If you still need to change colors, remember in [our docs](https://github.com/Laravel-Backpack/theme-tabler?tab=readme-ov-file#config) is the information to change as you want.

Cheers.

**Comment by jcastroa87** [comment]
Hello.

I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
